src/app/modules/terminal/components/terminal/terminal.component.ts

```diff
@@ -105,14 +105,17 @@ export class TerminalComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   initShell(): void {
-    this.authService.authToken$.pipe(
-      tap((token) => {
-        this.initializeWebShell(token);
-        this.shellService.shellOutput.pipe(untilDestroyed(this)).subscribe(() => {});
-        this.initializeTerminal();
-      }),
-      untilDestroyed(this),
-    ).subscribe();
+    setTimeout(() => {
```

Comment:
Why is this needed? We should avoid this kind of simple workarounds.

Reply:
yeah, sorry